integration-test/Css.test.tsx

```diff
@@ -288,6 +288,14 @@ describe("Css", () => {
       }
     `);
   });
+
+  it("has the palette", () => {
+    expect(Css.fill(Palette.Black).$).toMatchInlineSnapshot(`
```

Comment:
**praise:** This export is HUGE! We can now get rid of the HomeOwner Paletter in HOP 🎉